electrical/breakbeam.py

```diff
@@ -0,0 +1,90 @@
+import RPi.GPIO as GPIO
+import time
+from enum import Enum
+
+
+class Breakbeam:
+    ACCEPTABLE_TIME = 3 # time to wait to confirm a full break
+    class BEAM_PINS(Enum):
+        HALF1 = 17
+        HALF2 = 22
+        FULL1 = 23
+        FULL2 = 24
+
+    def __init__(self):
+        """
+        Initializes each sensor to detect changes. Initializes beam_broken to False.
+        Creates an empty list for the beams that are broken.
+        """
+        for BEAM_PIN in Breakbeam.BEAM_PINS:
+            GPIO.setmode(GPIO.BCM)
+            GPIO.setup(BEAM_PIN.value, GPIO.IN, pull_up_down=GPIO.PUD_UP)
+            GPIO.add_event_detect(BEAM_PIN.value, GPIO.BOTH, callback=self.break_beam_callback)
+        self.blocked_sensors = set()
```

Comment:
blocked_sensors is an attribute for your Breakbeam class - let's add this into the specification for _ _ init _ _ function